.deployment.config.json

```diff
@@ -81,6 +81,13 @@
         }
       }
     },
+    {
```

Comment:
This should be run only when doing a production release, and after the upload to s3 has happened.

I think with that config it would run all the time when we commit to master, since we create a dev package every time we merge.

And we certainly don't want that ;)